easybuild/easyconfigs/i/IQ-TREE/IQ-TREE-2.0.6-GCC-10.2.0.eb

```diff
@@ -0,0 +1,32 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CMakeMake'
+
+name = 'IQ-TREE'
+version = '2.0.6'
```

Comment:
Do we really need to go back to an old version here instead of 2.1.2 or later?

Reply:
I think there was some confusion, at least on my side:
https://github.com/Cibiv/IQ-TREE
When I submitted that, I was using EB-4.3.2. I noticed that the latest EB, 4.3.3 got IQ-TREE-2.1.2, which I could not find at the link above. However, coming from the home page (http://www.iqtree.org/) and following the link to the IQ-TREE-2 version, you are ending up at a different GitHub site which does contain the version you mentioned. 
All that happened in the time between me doing this PR and the latest EB release. 
You are right, we don't need the older version. At the time I was looking for the latest and only found the one for this PR.